core/controllers/custom_landing_pages.py

```diff
@@ -53,5 +54,6 @@ class StewardsLandingPage(base.BaseHandler):
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
-        self.render_template(
-            'pages/landing/stewards/landing_page_stewards.html')
+        (self.render_template(
```

Comment:
No need for outer parentheses, the inner ones will do. Here and elsewhere (also pointed by @seanlip)

Reply:
done